### PR TITLE
Add nginx config file to api Dockerfile volume

### PR DESCRIPTION
**Pull request by leandro2r**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,9 +13,8 @@ services:
     build: api/.
     env_file:
       - ./.env.example
```
Comment (kimuraz) [inline docker-compose.yml]: I guess here it's the same 😬

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -12,12 +12,14 @@ services:
   api:
     build: api/.
     env_file:
-      - ./.env
-    command: python ./manage.py runserver 0.0.0.0:8000
+      - ./.env.example
```
Comment (kimuraz) [inline docker-compose.dev.yml]: Shouldn't we use the `.env` file? There are some instructions on README.md to copy `.env.example` to `.env` to start using it, also the `.example` may suggest it wasn't supposed to be used, what do you think?

**Review by kimuraz — CHANGES_REQUESTED** [review]
Nice work :medal_military: 

Let me know your though about the comments :+1:

File: docker-compose.dev.yml (hunk 2 above)
Comment (leandro2r) [reply docker-compose.dev.yml]: My fault, I hadn't seen these instructions on the README.

File: docker-compose.yml (hunk 1 above)
Comment (leandro2r) [reply docker-compose.yml]: Agreed

**Review by kimuraz — APPROVED** [review]
:1st_place_medal:

**Pull request merged**